src/build_7_8/reports/StructuralVariantInterpretationLog.ts

```diff
@@ -0,0 +1,61 @@
+import { VariantValidation } from "./VariantValidation";
+import { User } from "./User";
+import { UserComment } from "./UserComment";
+import { StructuralVariantDetails } from "./StructuralVariantDetails";
+import { JsonObject, JsonProperty } from "json2typescript";
+
+@JsonObject
+export class StructuralVariantInterpretationLog {
+  /**
+  Variant details
+  */
+  @JsonProperty("variant", StructuralVariantDetails)
```

Comment:
minor comment but it would be nice to have spaces in between the fields like in `VariantInterpretationLog.ts` it makes it more readable

Reply:
Change made and pushed up